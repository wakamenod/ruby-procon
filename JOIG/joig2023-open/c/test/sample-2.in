3 4 100
116 194 258
57 155 222 360
