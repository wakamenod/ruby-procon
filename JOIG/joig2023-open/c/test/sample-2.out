41
61
72
0
